engine/src/main/java/org/hibernate/search/engine/spi/EntityIndexBinding.java

```diff
@@ -50,6 +50,18 @@
 	void postInitialize(Set<Class<?>> indexedClasses);
 
 	/**
+	 * @return the {@code IndexManager} impl used by this entity type
+	 */
+	Class<? extends IndexManager> getIndexManagerImpl();
+
+	/**
+	 * @param indexManagerCandidate the tested {@code IndexManager} implementation
+	 *
+	 * @return whether this entity is managed by the given {@code IndexManager} impl
+	 */
+	boolean isManagedBy(Class<? extends IndexManager> indexManagerCandidate);
```

Comment:
Seems this one can be avoided by just locally implementing it in the factories call this method. Tiny duplication, but prevents adding another SPI-level method.